### PR TITLE
Fix issue #181 (Bugfixing of Default Source and Default Stock)

### PR DESCRIPTION
**Pull request by aleron75**
This PR fixes the GetListTest for SourceRepository and StockRepository that were broken by the introduction of default source and stock

### CODE REVIEW

File: app/code/Magento/InventoryApi/Test/Api/SourceRepository/GetListTest.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ public function testGetList()
                 SearchCriteria::PAGE_SIZE => 2,
             ],
         ];
-        $expectedTotalCount = 4;
+        $expectedTotalCount = 5;
```
Comment (naydav) [inline app/code/Magento/InventoryApi/Test/Api/SourceRepository/GetListTest.php]: Default Stock and Default Source have created in `InventoryCatalog` 
but `InventoryApi` module has not dependency on `InventoryCatalog`

So we could not create tests which will be green only if `InventoryCatalog` module is enabled (logic of test has got dependency on `InventoryCatalog`)

**Review by naydav — CHANGES_REQUESTED** [review]

File: app/code/Magento/InventoryApi/Test/Api/SourceRepository/GetListTest.php (hunk 1 above)
Comment (aleron75) [reply app/code/Magento/InventoryApi/Test/Api/SourceRepository/GetListTest.php]: I understand; what's should be a correct approach in these cases?

Comment (naydav) [reply app/code/Magento/InventoryApi/Test/Api/SourceRepository/GetListTest.php]: I think we should not create sophisticated logic (we have more important tasks)
So we can use `assertGreatherThanOrEqual`

Comment (naydav) [reply app/code/Magento/InventoryApi/Test/Api/SourceRepository/GetListTest.php]: I already fixed

**Pull request merged**